praw/helpers.py

```diff
@@ -356,6 +356,28 @@ def b36_id(item):
             time.sleep(sleep_time)
 
 
+def chunk_sequence(sequence, chunk_length, allow_incomplete=True):
```

Comment:
Do you think it would be worthwhile to have unit tests to ensure this function works as expected?